frb/tests/test_associate.py

```diff
@@ -0,0 +1,111 @@
+import os
+import numpy as np
+from pkg_resources import resource_filename
+
+import pandas
+
+from astropy import units
+
+from frb.associate import frbassociate
+
+import pytest
+
+remote_data = pytest.mark.skipif(os.getenv('FRB_GDB') is None,
+                                 reason='test requires dev suite')
+
+@remote_data
+def test_individual():
+    from frb.associate import frbs
+    config = getattr(frbs, 'FRB180924'.lower())
+    frbA = frbassociate.run_individual(config)
+    # Test
+    assert isinstance(frbA.candidates, pandas.DataFrame)
+
+
+@remote_data
+def test_step_by_step():
```

Comment:
Not sure if the tests are also producing plots but those would be nice to have for visual checks/debugging.

Reply:
plots are real tough with CI